### ISSUE TITLE
Wonder

### ISSUE THREAD

**Issue by shanqiu24** [xinzhuma/monodle#33]
why when I train monodle：
(monodle) G:\WWProject\monodle\experiments\example>python ../../tools/train_val.py --config kitti_example.yaml
2022-10-11 10:46:15,326 INFO ################### Training ##################
2022-10-11 10:46:15,326 INFO Batch Size: 16
2022-10-11 10:46:15,326 INFO Learning Rate: 0.001250
Then it took more than half an hour to appear in the training epoch progress bar, I changed the original monodle to a single GPU run, mine is a single 3090
epochs: 0%| | 0/140 [00:00<?, ?it/s]
iters: 0%| | 0/232 [00:00<?, ?it/s]
